Mathlib/CategoryTheory/Sites/Hypercover/Zero.lean

```diff
@@ -98,6 +98,24 @@ lemma presieve₀_pullback₁ (f : S ⟶ T) (E : PreZeroHypercover.{w} T) [∀ i
   · rintro W g ⟨-, -, ⟨i⟩⟩
     use i
 
+/-- If `{Uᵢ}` covers `X`, this is the pre-`0`-hypercover of `X ×[Z] Y` given by `{Uᵢ ×[Z] Y}`. -/
+@[simps]
+noncomputable def pullbackCoverOfLeft {X Y Z : C} (f : X ⟶ Z) (g : Y ⟶ Z) [HasPullback f g]
```

Comment:
Would it make sense to make `f` and `g` appear after `E` so that we could use dot notation?
Also, it seems that this `0`-pre-hypercover is isomorphic to the `pullback₂` of `E` by the first projection from `pullback f g`. If this is correct, could you add this isomorphism? (and similarly for the right variant)
Would this isomorphism be helpful in order to obtain the `0`-hypercover variant?